<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Call Tree Studio</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="m_CentralWidget"/>
  <widget class="QMenuBar" name="m_Menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_MenuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_ActionNew"/>
    <addaction name="m_ActionOpen"/>
    <addaction name="m_ActionReload"/>
    <addaction name="m_ActionSave"/>
    <addaction name="m_ActionSaveAs"/>
   </widget>
   <widget class="QMenu" name="m_MenuHelp">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_ActionAboutQt"/>
    <addaction name="m_About"/>
   </widget>
   <widget class="QMenu" name="m_MenuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="m_ActionDelete"/>
   </widget>
   <widget class="QMenu" name="m_MenuNodes">
    <property name="title">
     <string>Nodes</string>
    </property>
    <addaction name="m_ActionSingleColumn"/>
    <addaction name="m_ActionMultiColumn"/>
   </widget>
   <addaction name="m_MenuFile"/>
   <addaction name="m_MenuEdit"/>
   <addaction name="m_MenuNodes"/>
   <addaction name="m_MenuHelp"/>
  </widget>
  <widget class="QDockWidget" name="m_NodesDock">
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Nodes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents1"/>
  </widget>
  <widget class="QDockWidget" name="m_PropertiesDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents2"/>
  </widget>
  <widget class="QToolBar" name="m_ToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_ActionNew"/>
   <addaction name="m_ActionOpen"/>
   <addaction name="m_ActionReload"/>
   <addaction name="m_ActionSave"/>
   <addaction name="m_ActionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="m_ActionSingleColumn"/>
   <addaction name="m_ActionMultiColumn"/>
  </widget>
  <action name="m_ActionOpen">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/page.png</normaloff>:/icons/page.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_ActionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="m_ActionSave">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/page_save.png</normaloff>:/icons/page_save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_ActionSaveAs">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/page_save_as.png</normaloff>:/icons/page_save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="toolTip">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="m_ActionDelete">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/bullet_delete.png</normaloff>:/icons/bullet_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="m_ActionNew">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/page_add.png</normaloff>:/icons/page_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_ActionReload">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/page_refresh.png</normaloff>:/icons/page_refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="m_ActionSingleColumn">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/application_view_list.png</normaloff>:/icons/application_view_list.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;ingle Column</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="m_ActionMultiColumn">
   <property name="icon">
    <iconset resource="../resources/cts_resources.qrc">
     <normaloff>:/icons/application_view_columns.png</normaloff>:/icons/application_view_columns.png</iconset>
   </property>
   <property name="text">
    <string>M&amp;ultiple Columns</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="m_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/cts_resources.qrc"/>
 </resources>
 <connections/>
</ui>
